### PR TITLE
Fix licensing status toggling

### PR DESCRIPTION
**Pull request by andrecardoso**
- Simplify authorization status listener
- Do not notify listener and send menssage when status is empty

### CODE REVIEW

**Comment by andrecardoso** [comment]
[(1) Defect 203: Licensed Chrome OS Displays sometimes report the Free version of Player is installed](https://trello.com/c/Rdw4HoHz/5124-1-defect-203-licensed-chrome-os-displays-sometimes-report-the-free-version-of-player-is-installed)

**Comment by andrecardoso** [comment]
@tejohnso right. I reverted it, please take a look at my last commit

File: src/licensing/index.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const productCodes = {
 
 const subscriptions = {};
 let displayId = null;
-let authorizationListeners = [];
+let authorizationStatusListener = () => {}; // eslint-disable-line func-style
```
Comment (tejohnso) [inline src/licensing/index.js]: Why is this no longer an array? Are we sure we're only going to have one listener?
If so, *notifyAuthorizationStatusListeners* should be renamed.

Comment (andrecardoso) [reply src/licensing/index.js]: Yes, there's always a single listener. I renamed the notify method. Thanks!

**Pull request merged**